walletSetup.ts

```diff
@@ -78,31 +81,21 @@ export function initInMemoryDataStorage({
     ),
     mt: new InMemoryMerkleTreeStorage(40),
 
-    states: new EthStateStorage(defaultEthConnectionConfig)
+    states: new EthStateStorage(conf)
   };
 
   return dataStorage;
 }
 
-export async function initMongoDataStorage({
-  rpcUrl,
-  contractAddress
-}: {
-  contractAddress: string;
-  rpcUrl: string;
-}): Promise<IDataStorage> {
+export async function initMongoDataStorage(): Promise<IDataStorage> {
   let url = mongoDbConnection;
   if (!url) {
     const mongodb = await MongoMemoryServer.create();
     url = mongodb.getUri();
   }
   const client = new MongoClient(url);
   await client.connect();
-  const db: Db = client.db('mongodb-sdk-example');
-
-  const conf: EthConnectionConfig = defaultEthConnectionConfig;
-  conf.contractAddress = contractAddress;
-  conf.url = rpcUrl;
+  const db: Db = client.db('mongodb- sdk-example');
```

Comment:
move the name to a constant?

Reply:
Sure! I'll actually move it to the env file